docs/docs.md

```diff
@@ -12,12 +12,23 @@ from dynamics import DynamicsClient
 - token_url: str - Url in form: 'https://[Dynamics Token URI]/path/to/token'
 - client_id: str - Client id (e.g. UUID).
 - client_secret: str - Client secret (e.g. OAuth password).
-- scope: List[str] - List of urls in form: 'https://[Organization URI]/scope'. Defines the database records that the API connection has access to.
+
+
+- *scope: Optional[str | List[str]] - Url or list of urls in form: 'https://[Organization URI]/scope'. Defines the database records that the API connection has access to.*
```

Comment:
You should use `Optional` syntax here, so separate values with commas, not pipes.